tests/scripts/basic-build-test.sh

```diff
@@ -52,6 +53,51 @@ export GNUTLS_SERV="$GNUTLS_SERV"
 CONFIG_H='include/mbedtls/config.h'
 CONFIG_BAK="$CONFIG_H.bak"
 
+EXITCODE=0
+
+LINE_COV_THRESHOLD=0
+FN_COV_THRESHOLD=0
+
+print_usage() {
+    echo "Usage: $0 [options]"
+    printf "  -h|--help\tPrint this help.\n"
+    printf "     --seed\tInteger seed value to use for this test run (default 1)\n"
+    printf "     --line-cov-threshold\tLine coveraged test threshold (default 0)\n"
```

Comment:
Minor: Shouldnt it be `coverage` instead of `coveraged`?

Reply:
Doh! Fixed.